client/src/themes/theme.js

```diff
@@ -1,14 +1,85 @@
 import {createMuiTheme} from '@material-ui/core';
 
+import Montserrat from 'typeface-montserrat';
+
 export const theme = createMuiTheme({
   typography: {
-    fontFamily: '"Roboto"',
-    fontSize: 12,
+    fontFamily: 'Montserrat, arial',
```

Comment:
We should ask the team if they want to adopt this font through out the rest of the project so we can expect to work on something that looks consistent. I do feel like leaving the default roboto font might be better through development. 

@ugur-akin 
@nicktran6997